src/scripts/pokemons/RoamingPokemonList.ts

```diff
@@ -0,0 +1,44 @@
+class RoamingPokemonList {
+    public static list: Array<Array<RoamingPokemon>> = [];
```

Comment:
```suggestion
    public static list: Partial<Record<GameConstants.Region, Array<RoamingPokemon>>> = {};
```
This suggested change isn't necessary, but is more specific about what can be used as object keys for this list.
The `Record<GameConstants.Region` ensures we are using a Region as the key, and the `Partial<..>` is just there so it doesn't complain when we haven't added something for every single region.

Feel free to ignore if you want to keep this as an Array :+1:

Reply:
That makes sense, and this way it also won't break if we happen to add any Pokemon to the `none` region for whatever reason.